src/python/registration.py

```diff
@@ -48,7 +60,7 @@ def _register_rom_configs(
 
                     # Register the environment
                     gymnasium.register(
-                        id=f"{prefix}{name}{flavour.suffix}-{config.version}",
+                        id=f"ALE/{prefix}{name}{flavour.suffix}-{config.version}",
```

Comment:
For v0 and v4 environments, there is no ALE prefix while v5 should have ALE prefix
Does this not add for all?